include/linux/thermal.h

```diff
@@ -259,7 +259,7 @@ static inline int thermal_generate_netlink_event(struct thermal_zone_device *tz,
 }
 #endif
 
-#ifdef CONFIG_AMLOGIC_THERMAL
+#ifdef CONFIG_AML_PLATFORM_THERMAL
 #include <mach/thermal.h>
 #endif /* CONFIG_AMLOGIC_THERMAL */
```

Comment:
Please change the comment after the above #endif to keep it consistent with corresponding #ifdef.